pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Web-crawling Framework.
+
+- Create a project:
+
+`scrapy startproject {{project_name}}`
+
+- Create a spider (in project directory):
+
+`scrapy genspider {{spider_name}} {{website_domain}}`
+
+- Edit spider (in project directory):
+
+`scrapy edit {{spider_name}}`
+
+- Run spider (in project directory):
+
+`scrapy crawl {{spider_name}}`
+
+- Fetch a webpage as scrapy sees it and print source in stdout:
+
+`scrapy fetch {{url}}`
+
+- View a webpage as scrapy sees it, opens it in default browser (recommended: to disable javascript beforehand):
```

Comment:
This wording still doesn't explain why one should "disable javascript beforehand". How about "Open a webpage in the default browser as scrapy sees it (disable javascript for extra fidelity)", or something like that?